src/prefect/cli/server.py

```diff
@@ -325,6 +325,9 @@ def start(
         )
         env.update(PREFECT_SERVER_DB_CMD=cmd)
 
+    # Pass the Core version so the Server API can return it
+    env.update(PREFECT_CORE_VERSION=prefect.__version__)
```

Comment:
This could also use `base_version[0]` from above where `base_version = prefect.__version__.split("+")` or we could use `default_tag` which is either `master` if there are extra commits or `core-VERSION` but it seems useful to know if the prefect launching the server is dirty/has extra commits. Hmm.

Reply:
Yea I agree, I like using `__version__` unprocessed 👍